scripts/ci_sanity_checks.sh

```diff
@@ -1,38 +1,62 @@
 #!/usr/bin/env bash
-set -x
-set -eu
+set -u
 
-# #############################################################################
+# #####################################################################################################################
 # This script runs on circle CI to verify that the code of the PR has been
 # sanitized before push.
-# #############################################################################
+#
+# WARNING: make sure you commit ALL your changes before running it locally if you ever do it because it will run a git
+# checkout -- which will reset your changes on all files...
+# #####################################################################################################################
+
+return_code=0
+
+# Check if any file has been modified. If yes, that means the best practices
+# have not been followed, so we will fail the job later but print a message here.
+check_diff(){
+  git diff --exit-code
+  code=$?
+  if [[ $code -ne 0 ]]; then
+    echo "[ERROR] You may need to do some cleanup in the files you commited, see the git diff output above."
+  fi
+  git checkout -- .
+  return_code=$(($return_code + $code))
+}
 
 # List the files that are different from the trunk
 from=$(git rev-parse refs/remotes/origin/trunk)
 to=$(git rev-parse HEAD)
 interval=${from}..${to}
 [[ "${from}" == "${to}" ]] && interval=${to}
 
-for f in $(git show -m --pretty="format:" --name-only ${interval}); do
+for f in $(git show -m --pretty="format:" --name-only ${interval} | sort | uniq); do
```

Comment:
sort | uniq --> sort -u

Reply:
Good catch!  I always forget that this switch is available! :)
Updated.